### PR TITLE
Update front-end

### PR DESCRIPTION
**Pull request by dntrng**
Update front-end params of recipes and ingredients to match with back-end

Update recipe model & add stub data for recipes & ingredients

Reformat code

### CODE REVIEW

File: client/src/components/login.jsx [hunk 1]
```diff
@@ -11,43 +11,41 @@ function Login() {
 	const [username, setUsername] = useState("");
 	const [password, setPassword] = useState("");
 	const [open, setOpen] = useState(false);
-	const [error,setError]=useState("");
-	
-	const [checkedLogin, setCheckedLogin] = useState(false);
```
Comment (dntrng) [inline client/src/components/login.jsx:16]: removed unused const

File: client/src/components/login.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import { Button, createTheme, CssBaseline, ThemeProvider, Box, Typography, Link} from "@mui/material";
-import React, { useState, useEffect } from "react";
```
Comment (dntrng) [inline client/src/components/login.jsx:2]: removed unused useEffect

File: client/src/tests/recipe.test.js [hunk 3]
```diff
@@ -1,97 +1,98 @@
 import { render } from '@testing-library/react';
 import { search } from 'components/searchBar';
```
Comment (davidndyck) [inline client/src/tests/recipe.test.js:2]: Certainly not worth rejecting the PR over, but it's interesting to note that some things are camelcase and some are snake.

File: docs/sequence-diagrams.md [hunk 4]
```diff
@@ -4,4 +4,8 @@ UML Sequence Diagrams for the COMP4350 W23 Group Project "PlatePlanner"
 
 ## Authentication:
 
-![Authentication Diagram](./authentication-diagram.png?raw=true "Diagram")
```
Comment (davidndyck) [inline docs/sequence-diagrams.md:7]: Whoops, good catch on the filename

**Review by davidndyck — APPROVED** [review]

**Comment by davidndyck** [comment]
Forgot to leave a comment, oops- Left one comment but not worth rejecting the PR over. 
Looks good and thanks for cleaning up all the formatting as well :)

**Pull request merged**